### PR TITLE
Milestone 4: Controllers

### PR DESCRIPTION
**Pull request by Rachelwebdev**
### In this milestone, I;

- Created a route, action and view file for each URL
- Added an HTML placeholders in the view files for each route
- Fixed linter errors. 

Here is a screenshot of my console

RAILS CONSOLE

### CODE REVIEW

File: config/routes.rb [hunk 1]
```diff
@@ -1,6 +1,9 @@
...
-  # Define your application routes per the DSL in https://guides.rubyonrails.org/routing.html
+ get 'users/:user_id/posts', to: 'posts#index', as: 'user_posts'
+ get 'users/:user_id/posts/:id', to: 'posts#show', as: 'user_post'
 
-  # Defines the root path route ("/")
-  # root "articles#index"
+ get 'users', to: 'users#index', as: 'users'
+ get 'users/:id', to: 'users#show', as: 'user'
+
+ root 'users#index'
```
Comment (Munsa1) [inline config/routes.rb]: - [ ] Its a requirement that we dont expose any unecessary endpoints. In order to acieve this we can use the `:only` option to specify only the routes you want to include:
e.g
```ruby
  resources :users, only: [:index, :show] do
    resources :posts, only: [:index, :show]
  end
```

Kindly fix this

**Review by Munsa1 — CHANGES_REQUESTED** [review]
Hi @Rachelwebdev, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>

## Highlights
Used Correct Naming Convention :heavy_check_mark: 
Descriptive PR :heavy_check_mark: 


## Required Changes ♻️
_Check the comments under the review._
- [ ] Please check the comment under review


## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by ichala — APPROVED** [review]
Hi Student  ,
# Status: Approved :heavy_check_mark: :heavy_check_mark: :heavy_check_mark:  
 
68747470733a2f2f6d65646961322e67697068792e636f6d2f6d656469612f6c3366516333614356304871707a5763552f67697068792e676966

# To Highlight 💪🏻

- Changes Implemented ✔️ 


 Good job implementing the requirements  :+1: Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

# Optional Suggestions  :

- N/A


**********
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._
Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag  ichala in your question so I can receive the notification.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by Rachelwebdev** [comment]
Thanks for the approval

**Pull request merged**